packages/terra-list/src/ListItem.jsx

```diff
@@ -54,13 +54,18 @@ const propTypes = {
    * @private Callback function not intended for use with this API, but if set pass it through to the element regardless.
    */
   onMouseDown: PropTypes.func,
+  /**
+   * Whether or not the list item is disabled.
```

Comment:
Based on my confusion above, lets enhance this doc to mention this will only be applied to selectable list items/ when `isSelectable` is defined.

Reply:
Updated in bc302c1